### PR TITLE
docs: refine statements for config customized spark parameters

### PR DESCRIPTION
**Pull request by amliuyong**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: docs/en/pipeline-mgmt/data-processing/configure-execution-para.md [hunk 1]
```diff
@@ -22,11 +22,9 @@ Syntax
 
 ## Config Spark job parameters
 
-By default, the EMR Serverless job is configured with [default settings][jobs-spark], which is suitable for most cases, e.g., hourly processing.
+By default, Clickstream evaluates the processing data set intelligently and set EMR job parameters appropriately. In most of time, you do not need to adjust the EMR job parameters, but if you want to override the EMR job parameters, you can put `spark-config.json` file in S3 bucket to set your job parameters.
```
Comment (luorobin-a2z) [inline docs/en/pipeline-mgmt/data-processing/configure-execution-para.md:25]: By default, the Clickstream pipeline automatically adjusts EMR job parameters based on the dataset volume that requires processing.

File: docs/zh/pipeline-mgmt/data-processing/configure-execution-para.md [hunk 2]
```diff
@@ -20,13 +20,9 @@
 
  ## 配置Spark作业参数
 
-默认情况下，EMR Serverless作业配置为[默认设置][jobs-spark]，适用于大多数情况，例如按小时处理。
+默认情况下，Clickstream 会智能地评估处理数据集并适当设置EMR作业参数。在大多数情况下，您不需要调整 EMR 作业参数，但如果您想覆盖 EMR 作业参数，您可以将 `spark-config.json` 文件放入S3存储桶以设置您自己的参数。
```
Comment (luorobin-a2z) [inline docs/zh/pipeline-mgmt/data-processing/configure-execution-para.md:23]: Clickstream 会智能地根据需要处理的数据集的大小并适当设置EMR作业参数

**Review by luorobin-a2z — APPROVED** [review]
with small changes

File: docs/en/pipeline-mgmt/data-processing/configure-execution-para.md (hunk 1 above)
Comment (amliuyong) [reply docs/en/pipeline-mgmt/data-processing/configure-execution-para.md]: changed.

File: docs/zh/pipeline-mgmt/data-processing/configure-execution-para.md (hunk 2 above)
Comment (amliuyong) [reply docs/zh/pipeline-mgmt/data-processing/configure-execution-para.md]: changed.

**Pull request merged**